OrientationPlugin.py

```diff
@@ -8,6 +8,9 @@
 from UM.Message import Message
 from cura.CuraApplication import CuraApplication
 
+from cura.CuraVersion import CuraVersion  # type: ignore
+from UM.Version import Version
```

Comment:
You forgot to use this import

Reply:
I've changed some things in the final result. Instead of using split, i actually use the version object to get the major version, as that is a lot more robust.